docs/contributor/assets/arch.drawio.svg

```diff
@@ -0,0 +1,529 @@
+<svg host="65bd71144e" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="1312px" height="721px" viewBox="-0.5 -0.5 1312 721" content="&lt;mxfile&gt;&lt;diagram id=&quot;wdlB93Bp5PP_D9yfYUkn&quot; name=&quot;Page-1&quot;&gt;7Vtdc+I2FP01zLQPYSx/wmMgsO0kaTPLtN19VGwB6toWI4sE9tdXsmVjW8IhQYR8NJnJoGtJSOfoHl1dOT1nnGy+ULha3pIIxT3bijY956pn28MA8L/CsC0MvuUVhgXFUWECO8MM/0TSaEnrGkcoa1RkhMQMr5rGkKQpClnDBiklj81qcxI3v3UFF0gxzEIYq9Z/cMSWhXVgBzv7bwgvluU3A39YPElgWVnOJFvCiDzWTM6k54wpIaz4lGzGKBbYlbgU7aZ7nlYDoyhlhzSwiwYPMF7LuX2BNEIpotx6A9MoC+EKybGybQkAJes0QqIPq+eMHpeYodkKhuLpI2ec25YsiXkJ8I8PiDLMwbuM8SLlNkZEhTmO4zGJCc17dEb5r7CTlNXsxQ+3Z4ySH6j2xM9/+JMFhRFGu1YpSflARnJm/MvRZi86oMKcr1VEEsTollcpGziSJrlOg3IBPtZYH0jbssa4XTaEcqUtqr53ZPAPkg89N47CzZ9s+T8xhRAMrX7Q4KZyqRo3jqfhxgkMcOMq3FxvE8gtY44SJaLqXQzTT0uQ41pPsuMFGnZczwA7XpeqzRCH/yhaWgTMPfH7IgL0BBvAv8K2VC5P4x1DDf7AN4C/34k/wfGHxx8MW/i7GvxtHf4m1Cnowv/y7vfj4DcAD3fzJjwDFZ7KjRvyYACdgYJOz/ZhImYZs3x+9dIMhRSxyrZo1ChLec17jq6fW2bXX/kX/FjfIx6BzvFCwZsPiMeq6CxYDxwVawA0S9GEEg+fh/VSBL7PgnocrzOWr+tbxGAEGTwr2I5/RrDLfj/zvucfoLsn2/cAUAgopIBr7gzRB8HDmZXXBy2ANMrrWLrAzAQ+6nGzSw7+IBF6nhrAkOEH3sZKiUgQnFMJ2kC/rhKoZ8cuoO8QzTCX0ZT9TeJ1gsYxxMnzkL8X61h8Y97BeaEP3AOg1x0+jEDvqnOPFmgmi4SyJVmQFMaTnXXUVIFdnRsihDHH5V/E2FamwuCakSZqaIPZN9G8b3uy+L326Goju84LW1koBipG1w00nwxZ01DWKqNWBukCsVrAr/JBUQxzh6z3dRS46sluili4FAvOKqQWhiHKRFEfC/DJ3sB7FDfhg3IfCvnEuUirG1SCo6hgCmX4J7zP+xMYrghOWT4jb9TzrroWsMxGysa9CsgG3m7nyrb6/GDtNhb3hSwejLXs/E4MvBYl+s1e/WYHZD7PRBTcoqoa4WHs+ed1jeAcrgEsPaPmfUM99bV9gy/eYrS2tc5EjvvducceoancA/hDu7GQzXjHazjH4JzOAWqeYfW9Tt/gbNDtt3rhe829RHnXLi+Zd6oiw3GEU+VNLymF21oFuVj3roKh14wrysPG9MD6jg9ai6QYwYuXjHq6nqE0ygpnyngcJiKyBCVEKsm7cvXBE67uuaCBrm3E0y9KULdliH4CX3ecTl+XaXjtHlbfwqwD3bTv1R0VvJ6X7glmjG99tpr4kNcxt7wrHAqPGJM45muOiFTIL9e3418VCj5aLsQNmrko19cc9XU5aNsycA6y1VyIvBuL30AaxLNbUq65n6oucevYmEgTlVL19rf6F2qBe4Id+2Bw1dQHRQuR3aBcBqYRqpUshlKYsne3NxYLqGtvdEDzdhyY2RxbVwkXZbLD6O64/3J9kj5gStIE5YMaCUUt5HwyerNyXq0JE6rlBv0mA7YmfQsCzcVZlWo8yre8swjXnkBmF7oYDWQCjXjtiUXNi5d6c66J6MeQr2PMxAj+SjHTJFvfun55T+iXHdhN/XKO06+ttoEZtVLTLpOru8+gRgAEyktYA7sUqLog6QIpUL7zeZS/qHf53CPmOHcTHvsjilP19v1zcKEL90/IhJqL+KpkG9uXRqviYjrPTeKU/4lhJtx1yduptbtusHB6UeY5um6t2jr5ipdSbuuNIG+oocc90YnDUQ/INXbYdqUhh2/DefKI9x2u1uIzXK1yd3oOMeULMG+UFd89gBXdW6RGWFGPyGqAxefGTreNi2grKwIvoIiXzEO1FHA6nfjjsSHZsltJPGcIFPx1mmXk/WpbgV+9u/3Q8A9bb8s4g0CzfZ8Mf/WcBz4X/m31cTX/X3Ay9NWrdDUt9aHRd1pv6PiB/3roqwc99U7gQ6Pvu34L/ZMpDy/u/qOqOOLt/i3NmfwH&lt;/diagram&gt;&lt;/mxfile&gt;">
```

Comment:
- If you don't adhere to our diagram guidelines for contributor docs, please at least use a white background so it renders well with all display settings.
- Use lower case for the connector descriptions ("sends", "fetches"). 
- Use lower case for "capacity units" or use "CUs" instead
- Why does number 1 have no connector description? 
- Why does "registers/deregisters tenant" have no number?

Reply:
Fixed